### ISSUE TITLE
Consider userscript

### ISSUE THREAD

**Issue by haarp** [Ibit-to/google-unlocked#14]
Hey,

since this gets taken down with glee, have you considered making it an userscript instead? These work through Greasemonkey/Tampermonkey/Violentmonkey/etc., but can often be installed directly like an "addon" aswell. This would make distribution and installation much easier (tons of sites for that) and work on any browser with an userscript engine, without needing specific addons for them.

Cheers!

**Comment by Ibit-to** [comment]
Good Idea! I will try to make a release. Also you are welcome to provide some examples or PR.

**Comment by heyjoeway** [comment]
Converted this to a userscript real quick as an example. (Not planning on maintaining it, just wanted to demonstrate that it's not a difficult conversion.) Tested in Tampermonkey on Chrome.
https://gist.github.com/heyjoeway/57f24ab5ce7b375d5b6846e9aa554bf3

**Comment by Ibit-to** [comment]
Will try to add an official one based on your example. Thank you!

**Comment by l0nax** [comment]
Any Updates @Ibit-to ?

**Comment by Ibit-to** [comment]
Did not have the time but everybody is welcome to send a PR!
